manageiq-ui-classic.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.executables   = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency "rails", "~> 5.0.0", ">= 5.0.0.1"
+  s.add_dependency "rails", ">= 5.0.0.1", "< 5.1"
```

Comment:
No; twiddle wakkas don't allow for prereleases of the next minor version. 

https://twitter.com/chrisarcand/status/835165159131840513